assets/data/cards_pt-br.json

```
@@ -37,9 +37,9 @@
     },
     {
       "title": "Ternário",
```

Aproveitando a alteração na descrição do termo, que tal se alterarmos o título para "Operador Ternário?

```suggestion
      "title": "Operador Ternário",
```